<launch>
  <arg name="veh"/>
  <arg name="node_name" value="encoder_localization_node" />

  <group ns="$(arg veh)">
    <node name="$(arg node_name)" pkg="encoder_localization" type="encoder_localization_node.py" output="screen">
    </node>
  </group>
</launch>
